operator/internal/controller/authorizationmodelrequest/authorizationmodelrequest_controller.go

```diff
@@ -206,6 +203,38 @@ func updateAuthorizationModelWithMissingInstances(
 	return true, nil
 }
 
+func getAuthorizationModelId(ctx context.Context,
+	openFgaService openfga.PermissionService,
+	modelRequestInstance extensionsv1.AuthorizationModelRequestInstance,
+	authorizationModelName string,
+	log *logr.Logger) (string, error) {
+	if modelRequestInstance.ExistingAuthorizationModelId != "" {
+		modelExists, err := openFgaService.CheckAuthorizationModelExists(ctx, modelRequestInstance.ExistingAuthorizationModelId)
+		if err != nil {
+			return "", err
```

Comment:
suggestion: wraps this error e.g. `fmt.Errorf("checking authorization model exists: %w", err)`

Reply:
Thanks - I have made an update :-)